lib/validation/validators.js

```diff
@@ -70,6 +70,9 @@ const validators = {
     }
     return isNull(value) || moment(value, 'YYYY-MM-DD').isBefore(isBefore);
   },
+  lessThanOrEqualToMaxWordCount(field, value, maxWordCount) {
+    return value?.split(/\s+/).filter(Boolean).length <= maxWordCount;
+  },
```

Comment:
Alternative to https://github.com/UKHomeOffice/asl-components/pull/352/files#r1850305521
```suggestion
  lessThanOrEqualToMaxWordCount(field, value) {
    return value?.split(/\s+/).filter(Boolean).length <= field.maxWordCount;
  },
```